src/locales/en.ts

```diff
@@ -35,6 +35,8 @@ const en = {
     correct: 'Correctly answered',
     question: 'Question',
     attempts: 'Attempts',
+    reference: 'Reference',
+    referenceLink: 'Review link',
```

Comment:
I find the wording 'Review link' a bit too technical. But I also don't really have a better idea for a wording at the moment. If we remove the heading as I suggested in my other comment it could also say "Review links".